src/docs/wtstats.dox

```diff
@@ -0,0 +1,33 @@
+/*! @page wtstats Visualizing performance with wtstats
+
+The WiredTiger distribution includes the \b wtstats tool that can be used to examine
+information generated using statistics logging (see @ref statistics_log).
+
+After running an application with statistics logging configured, the statistics output files will be in the database home directory.  By default, these are named \c WiredTigerStat.* .  In the database home directory, run this command, replacing \c ... with the path to the WiredTiger installation directory:
+\code{.sh}
+python .../tools/wtstats.py WiredTigerStat.*
```

Comment:
@ddanderson Do you want to go into any/all of the different args that `wtstats.py` can take in the documentation?  Also, it can take directory name(s) and it will process all WiredTigerStat files in the given directories.